actions/plugins/publish/promote-plugin/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Promote plugin
```

Comment:
Can we rename this to set plugin scope? Cause changing plugin scope is not actually promoting. It is more of a visibility thing. grafana_cloud: only on cloud, universal: all, grafana_cloud_org_{}: only on certain org.